samples/bluetooth/iso_broadcast_benchmark/src/receiver.c

```diff
@@ -94,6 +94,11 @@ static void scan_recv(const struct bt_le_scan_recv_info *info,
 		return;
 	}
 
+	if (info->interval == 0U) {
+		/* Not broadcast periodic advertising - Ignore */
+		return;
+	}
+
```

Comment:
Move it above line 89, no point in parsing data if we return here.

Reply:
Good point